lib/view_component/form/builder.rb

```diff
@@ -43,6 +44,7 @@ def check_box(method, options = {}, checked_value = "1", unchecked_value = "0")
       # def radio_button(method, tag_value, options = {})
       # end
 
+      # TODO: remove this method?
```

Comment:
@nicolas-brousse Is there something specific to do for `file_field`?

Reply:
Yes, I think we should do like here https://github.com/rails/rails/blob/83217025a171593547d1268651b446d3533e2019/actionview/lib/action_view/helpers/form_helper.rb#L2417.